legesher.yml

```
@@ -0,0 +1,12 @@
+---
+name: Up-For-Grabs.net
+desc: The bridge between the words we speak and the code we write.
+site: https://github.com/legesher/legesher
+tags:
+- html/css
+- ruby
+- python
+- javascript
+upforgrabs:
+  name: help needed with various projects
```

This field needs to match with the label used in the project:

```suggestion
  name: Good First Issue
```